mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/AttachmentLoader.java

```diff
@@ -63,6 +64,22 @@ public AttachmentLoader(CassandraAttachmentMapper attachmentMapper) {
         }
     }
 
+    public CompletableFuture<Stream<SimpleMessage>> addAttachmentToMessages(Stream<Pair<RawMessageWithoutAttachment,
```

Comment:
Here I introduce a big code duplication between the two addAttachmentToMessages IMO the first one should call the second one.